README.md

```diff
@@ -212,6 +212,19 @@ t = w1.transfer(0.00001, :eth, w2).wait!
 puts "Transfer successfully completed: #{t}"
 ```
 
+### Gasless USDC Transfers
+
+To transfer USDC without needing to hold ETH for gas, you can use the `transfer` method with the `gasless` option set to `true`.
```

Comment:
what is our intuition behind setting a gasless option vs defaulting to gasless for all transfers of USDC on base-mainnet?

Reply:
That is something that we can enable by default if we'd like, but initially we were planning on not changing the default behavior.

We can roll this out as a default on the backend (but would be a breaking change for clients that do not yet support this signing of sponsored send messages)